### DISCUSSION TITLE
Does anyone have qmediaplayer working?

### DISCUSSION THREAD

**Discussion by knoxvilledatabase** [elimintz/justpy#491] [Q&A]
Curious if anyone has got this working? https://github.com/quasarframework/quasar-ui-qmediaplayer

Thank you!

**Comment by WolfgangFahl** [comment]
@knoxvilledatabase you might want to ask this question at https://stackoverflow.com/questions/tagged/justpy and tag with quasar to to reach a bigger audience. justpy itself at this point has only 877 stars and 25 questions on stackoverflow while 
quasar has over 700 questions on stackoverflow https://stackoverflow.com/questions/tagged/quasar and https://quasar.dev/ with https://github.com/quasarframework/quasar has over 22k stars. 

Since justpy only is an integration platform and currently even doesn't support the latest version of quasar we need more support by the community in improving the justpy core. The questions and answers around the javascript frameworks supported need to be justpy specific before it makes sense to discuss them in detail here. Also the stackoverflow question and answer rules are much stricter and we'd love to enforce those stricter rules to be able to focus on development. So if your questions makes it thru the stackoverflow process and gets upvotes and answers it makes sense to come back here and discuss implementation details.

**Reply by knoxvilledatabase** [reply]
Right, I was curious if anyone on Github got QMediaPlayer working in Justpy. 

Thank you!
